app/src/androidTest/java/org/simple/clinic/storage/PurgeDatabaseAndroidTest.kt

```diff
@@ -410,4 +410,52 @@ class PurgeDatabaseAndroidTest {
     assertThat(appointmentDao.getOne(visitedButUnsyncedAppointment.uuid)).isEqualTo(visitedButUnsyncedAppointment)
     assertThat(appointmentDao.getOne(appointmentWithUnknownStatus.uuid)).isEqualTo(appointmentWithUnknownStatus)
   }
+
+  @Test
+  fun purging_the_database_should_delete_soft_patients_when_retention_time_has_passed() {
```

Comment:
wdym by `soft_patients`? Was it supposed to be `soft_delete`? 🤔

Reply:
It should be a hard delete. Was this just a copy-paste error?

Reply:
Yes it should be a hard delete, sorry. I got confused why all other tests were using soft delete. Should've asked, my bad.